src/evo.js

```diff
@@ -53,20 +53,9 @@ Telemetry.init(function() {
       $("#measure").change(function() {
         // Update the measure description
         var measure = $("#measure").val();
-        var measureEntry = gMeasureMap[measure];
-        $("#measure-description").text(measureEntry.description + " (" + measure + ")");
+        var description = gMeasureMap[measure].description;
+        $("#measure-description").text(description + " (" + measure + ")");
         $("#submissions-title").text(measure + " submissions");
-        
-        // Figure out which aggregates actually apply to this measure
-        var options;
-        if (measureEntry.kind == "linear" || measureEntry.kind == "exponential") {
-          options = [["median", "Median"], ["mean", "Mean"], ["5th-percentile", "5th Percentile"], ["25th-percentile", "25th Percentile"], ["75th-percentile", "75th Percentile"], ["95th-percentile", "95th Percentile"]]
-        } else {
-          options = [["submissions", "Submissions"]];
-        }
```

Comment:
I think we still want this. Mean makes sense for flag, boolean histograms, the percentiles don't. Similarly it makes no sense to show Evo chart for enumerated histograms for example